skbeam/core/mask.py

```diff
@@ -79,7 +79,7 @@ def bad_to_nan_gen(images, bad):
             yield im
 
 
-def threshold_mask(images, threshold, mask=None):
```

Comment:
Actually I was thinking of removing the streaming aspect to it, but I didn't write it so I didn't want to step on anyone's toes. (I fixed the PEP8 but not the deputy)